src/System.Private.ServiceModel/tools/test/SelfHostWcfService/TestResources/WebSocketResources.cs

```diff
@@ -47,4 +42,54 @@ protected override Binding GetBinding()
             return binding;
         }
     }
+
+    internal class WebSocketHttpDuplexBinaryStreamed : EndpointResource<WSDuplexService, IWSDuplexService>
+    {
+        protected override string Protocol { get { return BaseAddressResource.Http; } }
+
+        protected override int GetPort(ResourceRequestContext context)
+        {
+            return context.BridgeConfiguration.BridgeWebSocketPort;
+        }
+
+        protected override string Address { get { return "WebSocketHttpDuplexBinaryStreamed"; } }
+
+        protected override Binding GetBinding()
+        {
+            int DefaultMaxReceivedMessageSize = 64 * 1024 * 1024;
```

Comment:
Nit: this number keeps showing up, so maybe it's time to go into a Constants file somewhere.